cmd/status/printers/table/adapter.go

```diff
@@ -52,13 +56,16 @@ func (rss *ResourceState) Error() error {
 	return rss.err
 }
 
-func (ca *CollectorAdapter) LatestStatus() *ResourceState {
+func (ca *CollectorAdapter) LatestStatus(invNameMap map[object.ObjMetadata]string, statusSet map[string]bool) *ResourceState {
```

Comment:
Do we have to provide this as parameters to this function? I don't think these are changed as part of the printing process, so it seems like it could be passed in when we create the `CollectorAdapter`?

Reply:
Yeah that makes sense, they do not need to be passed as parameters. Thanks for pointing it out, I will update this.